cue.go

```diff
@@ -108,7 +108,7 @@ func CueValidate(query, cueFile, currentPath string) (tc CanBeAPart, err error)
 		logicalOperation := t.Validate(rootValue, CuePath{}, blockedRootFields)
 		tc = logicalOperation
 		if logicalOperation.Error != nil {
-			err = fmt.Errorf(*logicalOperation.Error)
+			err = fmt.Errorf("%s", *logicalOperation.Error)
```

Comment:
We actually don't need fmt.Errorf at all here. 
```suggestion
			err = *ptc.Error
```

Reply:
logicalOperation.Error is also *string

Reply:
No worries.